wp-content/plugins/core/src/Assets/Admin/Scripts.php

```diff
@@ -52,4 +44,5 @@ private function localize_scripts( string $handle ): void {
 		wp_localize_script( $handle, 'modern_tribe_admin_i18n', $this->localization->get_data() );
```

Comment:
Ran acceptance tests locally and every test failed from this error:
```
TypeError: Argument 1 passed to Tribe\Project\Assets\Admin\Scripts::localize_scripts() must be of the type string, bool given, called in /application/www/wp-content/plugins/core/src/Assets/Admin/Scripts.php on line 40 in file /application/www/wp-content/plugins/core/src/Assets/Admin/Scripts.php on line 43
Stack trace:
  1. TypeError-&gt;() /application/www/wp-content/plugins/core/src/Assets/Admin/Scripts.php:43
  2. Tribe\Project\Assets\Admin\Scripts-&gt;localize_scripts() /application/www/wp-content/plugins/core/src/Assets/Admin/Scripts.php:40
  3. Tribe\Project\Assets\Admin\Scripts-&gt;enqueue_scripts() /application/www/wp-content/plugins/core/src/Assets/Assets_Subscriber.php:62
  4. Tribe\Project\Assets\Assets_Subscriber-&gt;Tribe\Project\Assets\{closure}() /application/www/wp/wp-includes/class-wp-hook.php:290
  5. WP_Hook-&gt;apply_filters() /application/www/wp/wp-includes/class-wp-hook.php:316
  6. WP_Hook-&gt;do_action() /application/www/wp/wp-includes/plugin.php:484
  7. do_action() /application/www/wp/wp-admin/admin-header.php:102
  8. require_once() /application/www/wp/wp-admin/term.php:73
 ```

Reply:
You don't have the core theme active but the core plugin.

Reply:
Cleared the DB same issue. I was also under the impression that it does a fresh import for every test and should properly activate the theme and plugins required for the test.

Reply:
It should, not sure if it activates the theme though. I'll fix the bad typing on this this though.

Reply:
Pull and give that a try

Reply:
Passes after these updates and my own local config issue was fixed.